backend/django_core/static_src/components/page/explore/Index.svelte

```diff
@@ -0,0 +1,220 @@
+            />
+        </div>
+    </div>
+
+    <div class="mt-3 flex items-end justify-between gap-3 md:mt-0 md:gap-0">
+        <div class="flex items-center gap-3 md:gap-[1.5vw]">
+            <div class="hidden flex-col gap-[0.35vw] md:flex">
+                <span class="text-[1vw] font-semibold leading-none">Search Animes</span>
+                <div class="relative flex items-center">
+                    <div class="absolute md:ml-[1vw]"><coreproject-icon-search class="md:w-[1.1vw]"></coreproject-icon-search></div>
+                    <input
+                        type="text"
+                        placeholder="Looking for specific anime? Start from here..."
+                        class="w-[30vw] rounded-[0.5vw] border-none bg-neutral py-[0.8vw] pl-[3vw] text-[1vw] leading-none focus:ring-0 md:bg-neutral text-neutral-content font-semibold placeholder:text-neutral-content/75 placeholder:font-medium"
+                    />
+                </div>
+            </div>
+            {#each Object.entries(filter_options_mapping) as option}
+                {@const title = option[1].title}
+                {@const klass = option[1].class}
+                {@const selected_items = option[1].selected_items}
+                {@const filter_items = option[1].items}
+
+                <div class={cn(klass, "group dropdown dropdown-bottom")}>
+                    <span class="font-semibold leading-none md:text-[1vw]">{title}</span>
+                    <div class="relative flex items-center">
+                        <span class="absolute flex items-center md:gap-[0.25vw] cursor-pointer opacity-100 duration-300 group-focus-within:opacity-0">
+                            {#if selected_items.length > 0}
+                                <span class="ml-3 badge badge-primary rounded p-1 text-sm font-semibold md:ml-[0.75vw] md:rounded-[0.25vw] md:p-[0.35vw] md:text-[0.85vw] md:h-[1.5vw]">
+                                    <!-- show first item -->
+                                    {selected_items[0][1]}
+                                </span>
+                                <!-- show count of remaining items if exists -->
+                                {#if selected_items.length > 1}
+                                    <span class="ml-1 rounded badge md:h-[1.5vw] p-1 text-sm font-semibold md:ml-[0.15vw] md:rounded-[0.25vw] md:p-[0.35vw] md:text-[0.85vw]">
+                                        +{selected_items.filter((item) => item !== selected_items[0]).length}
+                                    </span>
+                                {/if}
+                            {:else}
+                                <span class="ml-3 text-base md:ml-[1vw] md:text-[0.9vw]">Any</span>
+                            {/if}
+                        </span>
+                        <input
+                            bind:value={option[1].value}
+                            on:blur={() => (option[1].value = "")}
```

Comment:
This should be a `<svelte:window/>` modifier

Reply:
No? this is supposed for clearing value when click outside that specific input.
Not on:blur window